app/views/pages/home.html.erb

```diff
@@ -0,0 +1,10 @@
+<a href="/plants" class="myButton">lista roślinek</a>
```

Comment:
Let's use the [`link_to` helper](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) here:
```suggestion
<%= link_to 'lista roślinek", plants_path, class: "myButton" %>
<a href="/plants" class="myButton">lista roślinek</a>
```